PlotSquared-Sponge-17.11.18-f27b12a Mysql init fault

# Bug report template 
<!--- In order to create a valid issue report you have to follow this template. -->
<!--- Incomplete reports might be marked as invalid. -->
<!--- You may remove it if you are posting a feature request. -->
**Debug paste link:** 
<!--- Enter /plot debugpaste in game or in your console and copy the output here -->
https://hastebin.com/dazoliqice

**Description of the problem:** 
The command '/plot database mysql' results in console error. Latest sponge and plotsquared used for testing. I'll try other versions just in case.

`[04:04:34 ERROR] [STDERR]: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
[04:04:34 ERROR] [STDERR]:      at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
[04:04:34 ERROR] [STDERR]:      at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[04:04:34 ERROR] [STDERR]:      at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[04:04:34 ERROR] [STDERR]:      at java.lang.Class.forName0(Native Method)
[04:04:34 ERROR] [STDERR]:      at java.lang.Class.forName(Class.java:264)
[04:04:34 ERROR] [STDERR]:      at com.intellectualcrafters.plot.database.MySQL.forceConnection(MySQL.java:46)
[04:04:34 ERROR] [STDERR]:      at com.intellectualcrafters.plot.database.MySQL.openConnection(MySQL.java:57)
[04:04:34 ERROR] [STDERR]:      at com.intellectualcrafters.plot.database.SQLManager.<init>(SQLManager.java:107)
[04:04:34 ERROR] [STDERR]:      at com.intellectualcrafters.plot.commands.Database.onCommand(Database.java:169)
[04:04:34 ERROR] [STDERR]:      at com.intellectualcrafters.plot.commands.SubCommand.execute(SubCommand.java:27)
[04:04:34 ERROR] [STDERR]:      at com.plotsquared.general.commands.Command.execute(Command.java:337)
[04:04:34 ERROR] [STDERR]:      at com.intellectualcrafters.plot.commands.MainCommand.execute(MainCommand.java:260)
[04:04:34 ERROR] [STDERR]:      at com.intellectualcrafters.plot.commands.MainCommand.onCommand(MainCommand.java:135)
[04:04:34 ERROR] [STDERR]:      at com.plotsquared.sponge.util.SpongeCommand.lambda$process$0(SpongeCommand.java:41)
[04:04:34 ERROR] [STDERR]:      at org.spongepowered.api.scheduler.Task$Builder.lambda$execute$0(Task.java:139)
[04:04:34 ERROR] [STDERR]:      at org.spongepowered.common.scheduler.SchedulerBase.lambda$startTask$0(SchedulerBase.java:187)
[04:04:34 ERROR] [STDERR]:      at org.spongepowered.common.scheduler.SyncScheduler.executeTaskRunnable(SyncScheduler.java:81)
[04:04:34 ERROR] [STDERR]:      at org.spongepowered.common.scheduler.SchedulerBase.startTask(SchedulerBase.java:180)
[04:04:34 ERROR] [STDERR]:      at org.spongepowered.common.scheduler.SchedulerBase.processTask(SchedulerBase.java:166)
[04:04:34 ERROR] [STDERR]:      at java.util.concurrent.ConcurrentHashMap$ValuesView.forEach(ConcurrentHashMap.java:4707)
[04:04:34 ERROR] [STDERR]:      at org.spongepowered.common.scheduler.SchedulerBase.runTick(SchedulerBase.java:109)
[04:04:34 ERROR] [STDERR]:      at org.spongepowered.common.scheduler.SyncScheduler.tick(SyncScheduler.java:51)
[04:04:34 ERROR] [STDERR]:      at org.spongepowered.common.scheduler.SpongeScheduler.tickSyncScheduler(SpongeScheduler.java:185)
[04:04:34 ERROR] [STDERR]:      at net.minecraft.server.dedicated.DedicatedServer.handler$onTick$zof000(SourceFile:1254)
[04:04:34 ERROR] [STDERR]:      at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(SourceFile:337)
[04:04:34 ERROR] [STDERR]:      at net.minecraft.server.MinecraftServer.func_71217_p(SourceFile:562)
[04:04:34 ERROR] [STDERR]:      at net.minecraft.server.MinecraftServer.run(SourceFile:466)
[04:04:34 ERROR] [STDERR]:      at java.lang.Thread.run(Thread.java:748)
[04:04:34 ERROR] [STDERR]: Caused by: java.lang.NullPointerException`

Current version as visible in debug:
spongevanilla-1.11.2-6.1.0-BETA-22
PlotSquared-Sponge-17.11.18-f27b12a-1938


**How to replicate:** 
Launch a server with PlotSquared currently set with sqlite
Run '/plot database mysql .... ' in console

**Checklist**:
<!-- Make sure you have completed the following steps (put an "X" between of brackets): -->
- [X] I included a `/plot debugpaste` link
- [X] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/PlotSquared/issues?utf8=%E2%9C%93&q=is%3Aissue)
- [X] I made sure I am using an up-to-date version of PlotSquared
- [X] I Made sure the bug/error is not caused by any other plugin

I have the same issue, with this stacktrace:

```
[19:08:02] [Server thread/INFO] [plotsquared]: [P2] &cFailed to open DATABASE connection. The plugin will disable itself.
[19:08:02] [Server thread/INFO] [plotsquared]: $4MYSQL
[19:08:02] [Server thread/INFO] [plotsquared]: &d==== Here is an ugly stacktrace, if you are interested in those things ===
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: java.lang.ClassNotFoundException: com.mysql.jdbc.Driver
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at java.lang.Class.forName0(Native Method)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at java.lang.Class.forName(Class.java:264)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.intellectualcrafters.plot.database.MySQL.forceConnection(MySQL.java:46)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.intellectualcrafters.plot.database.MySQL.openConnection(MySQL.java:57)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.intellectualcrafters.plot.database.SQLManager.<init>(SQLManager.java:107)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.intellectualcrafters.plot.PS.setupDatabase(PS.java:1607)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.intellectualcrafters.plot.PS.<init>(PS.java:163)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.plotsquared.sponge.SpongeMain.onServerAboutToStart(SpongeMain.java:145)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at org.spongepowered.common.event.listener.GameAboutToStartServerEventListener_SpongeMain_onServerAboutToStart42.handle(Unknown Source)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at org.spongepowered.common.event.RegisteredListener.handle(RegisteredListener.java:95)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:335)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:368)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at org.spongepowered.common.SpongeImpl.postEvent(SpongeImpl.java:213)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at org.spongepowered.mod.SpongeMod.onStateEvent(SpongeMod.java:240)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:278)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:256)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at net.minecraftforge.fml.common.LoadController.redirect$onPost$zza000(LoadController.java:560)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:148)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at net.minecraftforge.fml.common.Loader.serverAboutToStart(Loader.java:842)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at net.minecraftforge.fml.common.FMLCommonHandler.handleServerAboutToStart(FMLCommonHandler.java:291)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:268)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:484)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:setupDatabase:1628]: 	at java.lang.Thread.run(Thread.java:748)
[19:08:02] [Server thread/INFO] [STDERR]: [java.lang.Throwable:printStackTrace:634]: Caused by: java.lang.NullPointerException
[19:08:02] [Server thread/INFO] [plotsquared]: &d==== End of stacktrace ====
[19:08:02] [Server thread/INFO] [plotsquared]: &6Please go to the PlotSquared 'storage.yml' and configure the database correctly.
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: java.lang.UnsupportedOperationException: NOT IMPLEMENTED YET
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.plotsquared.sponge.util.KillRoadMobs.run(KillRoadMobs.java:6)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.plotsquared.sponge.SpongeMain.runEntityTask(SpongeMain.java:290)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.intellectualcrafters.plot.PS.<init>(PS.java:169)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.plotsquared.sponge.SpongeMain.onServerAboutToStart(SpongeMain.java:145)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at org.spongepowered.common.event.listener.GameAboutToStartServerEventListener_SpongeMain_onServerAboutToStart42.handle(Unknown Source)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at org.spongepowered.common.event.RegisteredListener.handle(RegisteredListener.java:95)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:335)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at org.spongepowered.mod.event.SpongeModEventManager.post(SpongeModEventManager.java:368)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at org.spongepowered.common.SpongeImpl.postEvent(SpongeImpl.java:213)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at org.spongepowered.mod.SpongeMod.onStateEvent(SpongeMod.java:240)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at net.minecraftforge.fml.common.LoadController.sendEventToModContainer(LoadController.java:278)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at net.minecraftforge.fml.common.LoadController.propogateStateMessage(LoadController.java:256)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.Subscriber.invokeSubscriberMethod(Subscriber.java:91)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.Subscriber$SynchronizedSubscriber.invokeSubscriberMethod(Subscriber.java:150)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.Subscriber$1.run(Subscriber.java:76)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.util.concurrent.MoreExecutors$DirectExecutor.execute(MoreExecutors.java:399)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.Subscriber.dispatchEvent(Subscriber.java:71)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.Dispatcher$PerThreadQueuedDispatcher.dispatch(Dispatcher.java:116)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at com.google.common.eventbus.EventBus.post(EventBus.java:217)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at net.minecraftforge.fml.common.LoadController.redirect$onPost$zza000(LoadController.java:560)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at net.minecraftforge.fml.common.LoadController.distributeStateMessage(LoadController.java:148)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at net.minecraftforge.fml.common.Loader.serverAboutToStart(Loader.java:842)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at net.minecraftforge.fml.common.FMLCommonHandler.handleServerAboutToStart(FMLCommonHandler.java:291)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:268)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:484)
[19:08:02] [Server thread/INFO] [STDERR]: [com.intellectualcrafters.plot.PS:<init>:303]: 	at java.lang.Thread.run(Thread.java:748)
```

--- a/Core/src/main/java/com/intellectualcrafters/plot/database/MySQL.java
+++ b/Core/src/main/java/com/intellectualcrafters/plot/database/MySQL.java
@@ -42,8 +42,7 @@ public class MySQL extends Database {
     }
 
     @Override
-    public Connection forceConnection() throws SQLException, ClassNotFoundException {
-        Class.forName("com.mysql.jdbc.Driver");
+    public Connection forceConnection() throws SQLException {
         this.connection =
                 DriverManager.getConnection("jdbc:mysql://" + this.hostname + ':' + this.port + '/' + this.database + "?" + StringMan.join(Storage.MySQL.PROPERTIES, "&"), this.user, this.password);
         return this.connection;

